index.js

```diff
@@ -1,18 +1,42 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector('h1#header')
 console.log("Here's your header:", header)
 
+// I changed the argument for querySelector so that the h1 tag was included.
+// I also added defer attribute to the html file so that the the script file is executed only after the HTML file has loaded.
 
 /***** Deliverable 2 *****/
 
+header.style.color = 'red'
 
+// I just googled how to use JavaScript to alter an elements color, and this was the first solution that came up.
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const travelerName = document.querySelector("h2");
+travelerName.innerText = traveler.name;
 
+const travelerNickname = document.querySelector("em");
+travelerNickname.innerText = traveler.nickname;
+
+const travelerImage = document.querySelector("img");
+travelerImage.src = traveler.photo;
+travelerImage.alt = traveler.name;
```

Comment:
Good! You found the elements that you were wanting to update, then updated them!